sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -116,7 +116,8 @@ def _subprocess_command(cls, job_port, expansion_port):
       tmp_dir = mkdtemp(prefix='flinktest')
 
       cls._create_conf_dir()
-      cls.expansion_port = expansion_port
+      # TODO Move expansion address into PipelineOptions
```

Comment:
Not sure this makes sense as a pipeline option, there may be different services to use for different transforms.

Reply:
I copied over this TODO. We discussed a while ago that adding a registry in pipeline options for expansion services might make sense.